addons/base_import/models/base_import.py

```diff
@@ -1188,8 +1188,8 @@ def _parse_import_data_recursive(self, model, prefix, data, import_fields, optio
     def _parse_date_from_data(self, data, index, name, field_type, options):
         dt = datetime.datetime
         fmt = fields.Date.to_string if field_type == 'date' else fields.Datetime.to_string
-        d_fmt = options.get('date_format')
-        dt_fmt = options.get('datetime_format')
+        d_fmt = options.get('date_format') or DEFAULT_SERVER_DATE_FORMAT
```

Comment:
here since the issue seems to be only about datetime, I don't know if this part is necessary